04 - Guides, Workflows, & Courses/Guides/Best Practices and Tips for Theme Development.md

```diff
@@ -19,6 +19,9 @@ This note serves as a collection of tips and best practices for theme developmen
 		- **Caution:** directly using the `.element.svelte-xyz` type of classes in a CSS snippet will make it stop working if there's a change to the plugin, and so it's best to not include the `.svelte-xyz` parts themselves (so just the element class), and then add more specificity if required. The `.svelte-xyz` classes are randomly generated, they work in the original style because they're being added through a variable. Always only target classes that look like they were made by a human (even if that may be `.hd-2`).
 	- You could also achieve more specificity by adding more elements before the targeted class, like `body .workspace-leaf-content .timer-start-button`, but an advantage of `body .timer-start-button.timer-start-button` is that it's not dependent on any other classes - if `.workspace-leaf-content` changed at some point, or the timer start button was used outside it, the custom style would still work if you just doubled its class.
 - You can enter `process.versions.chrome` in the console to get the current Chrome version utilized by Obsidian to render CSS. CSS documentation pages like [MDN](https://developer.mozilla.org/en-US/) or [W3-Schools](https://www.w3schools.com/cssref/css3_pr_overflow-y.asp) usually document which Chrome version is required for a certain CSS feature at the bottom of the feature's documentation page.
+- An easy way to get both a style settings class toggle and a regular cssclass is to either:
+	- Avoid specifying the view and just using a non-specific `.class-name` styling
```

Comment:
I don't understand what this sub-bullet means.

Reply:
I guess I mean something like avoiding this: `.class-name .markdown-preview-view .selector` so that `.class-name .selector` would easily make it a csscclass and a toggle without needing to do the other bullet below this 🤔